kuma/static/styles/minimalist/structure/_article.scss

```diff
@@ -20,5 +20,10 @@ Main wiki article content
         margin-top: 32px;
         font-size: 0.88889rem;
         color: $grey;
+
+        .contributors-sub-label {
+            margin: 5px;
+            font-weight: bold;
+        }
```

Comment:
You can replace the above with the following:

```
.icon-clock {
    vertical-align: unset;
}

header {
    display: inline-block;
    padding: 4px 0;
}
```

Reply:
Would you be alright with the header padding just being `4px` so that it gives a little space between the elements?

![Screen Shot 2019-10-10 at 9 47 12 AM](https://user-images.githubusercontent.com/1201062/66574862-fd352c00-eb42-11e9-9d55-4ed1dcdef3c6.png)

Reply:
With `padding: 4px;`

![Screen Shot 2019-10-10 at 9 55 01 AM](https://user-images.githubusercontent.com/1201062/66575558-1b4f5c00-eb44-11e9-99c4-f74d10d0b3e1.png)